testapps/test_pillowtop/tests/data/all-pillow-meta.json

```diff
@@ -219,7 +219,7 @@
     "UpdateUserSyncHistoryPillow": {
         "advertised_name": "UpdateUserSyncHistoryPillow",
         "change_feed_type": "KafkaChangeFeed",
-        "checkpoint_id": "synclog-user-sync",
+        "checkpoint_id": "update-user-sync-history",
```

Comment:
confused why this isn't `UpdateUserSyncHistoryPillow`

Reply:
I was trying to reverse guess the logic by looking at other pillow names in this file. I assumed there might be a function that coverts class names like `UpdateUserSyncHistoryPillow` to `update-user-sync-history`. But it doesn't look like so. I have updated